projects/fal/src/fal/app.py

```diff
@@ -91,7 +93,7 @@ def collect_routes(self) -> dict[RouteSignature, Callable[..., Any]]:
         }
 
     @asynccontextmanager
-    async def lifespan(self, app: FastAPI):
+    async def lifespan(self, app: "FastAPI"):
```

Comment:
I think you can keep it without quotes because it is still being imported i type checking mode